pkg/kubelet/dockershim/docker_stats_windows.go

```diff
@@ -52,6 +52,14 @@ func (ds *dockerService) getContainerStats(containerID string) (*runtimeapi.Cont
 
 	stats, err := hcsshimContainer.Statistics()
 	if err != nil {
+		if strings.Contains(err.Error(), "0x5") || strings.Contains(err.Error(), "0xc0370105") {
```

Comment:
Don't forget to import "strings"

Reply:
Had it staged but not committed. guess I missed it when I committed the patch :smile: